Neos.ContentRepository.Core/Classes/Feature/NodeModification/NodeModification.php

```diff
@@ -152,7 +150,7 @@ private function splitPropertiesToUnsetByScope(PropertyNames $propertiesToUnset,
         }
 
         return array_map(
-            fn(array $propertyValues): PropertyNames => PropertyNames::fromArray($propertyValues),
+            static fn(array $propertyValues): PropertyNames => PropertyNames::fromArray($propertyValues),
```

Comment:
not related to your change, but since you adjusted this line I would suggest to change this to:
```suggestion
            PropertyNames::fromArray(...),
```